src/README.md

```diff
@@ -0,0 +1,80 @@
+# iGym
+iGym is a mobile-first website designed to help users track and manage their gym workouts. The app stores all essential workout information, allowing users to focus on training without worrying about workout routines or weights.
+
+## Features
+- Organize and track your daily workouts.
+- Record weights, repetitions, and other workout details.
+- Seamlessly access your workout data from anywhere.
+
```

Comment:
I think one advantage of our application facing others in the market is that our application allows the user to get registered in various gyms having one workout plan in each gym